cli/src/test/java/com/devonfw/tools/ide/tool/jmc/JmcTest.java

```diff
@@ -25,47 +31,54 @@ public class JmcTest extends AbstractIdeContextTest {
 
   private static WireMockServer server;
 
-  private static Path resourcePath = Path.of("src/test/resources");
+  private static final Path RESOURCE_PATH = Path.of("src/test/resources");
 
   @BeforeAll
-  static void setUp() throws IOException {
+  static void setUp() {
+
+    // TODO use random port number and create url file dynamically in project
+    // TODO ISSUE:https://github.com/devonfw/IDEasy/issues/223
     server = new WireMockServer(WireMockConfiguration.wireMockConfig().port(1112));
```

Comment:
Why dont we remove wiremock now? I thought this is the entire point of the refactoring and the new test infrastructure.